src/api/Shrooms.Premium/Presentation/WebViewModels/Events/CreateEventViewModel.cs

```diff
@@ -18,16 +19,17 @@ public class CreateEventViewModel
         public string ImageName { get; set; }
 
         [Required]
-        public DateTime StartDate { get; set; }
```

Comment:
Why these made `nullable`, it seems a bit strange that they have `Required` attribute, but you still except `null` values

Reply:
[Required] applied on struct or a value type does not work, because they have default values. For example, if I don't provide a value for **StartDate** in my POST request, when it hits the controller method it will have a value that will be set, regardless if it exists in the request or not and because of that ModelState.IsValid will return **true** instead of false. Since, there was [Required] added previously, I thought that we are not supposed to allow such behaviour.